code/game/atoms_movable.dm

```diff
@@ -655,64 +638,13 @@
 */
 
 /atom/movable/proc/set_stationloving(state, inform_admins=FALSE)

```

Comment:
I feel like this should be removed now, if an admin wants to use this feature they can add the component themselves or remove it from the component list.